myapp/handlers/matches-handler.go

```diff
@@ -153,9 +153,14 @@ func (h *Handlers) Matches(w http.ResponseWriter, r *http.Request) {
 	userID := h.App.Session.GetInt(r.Context(), "userID")
 	userSpotTokens, err := h.Models.SpotifyTokens.GetSpotifyTokenForUser(userID)
 	if err != nil {
-		fmt.Println("Error getting spotify token", err)
+		fmt.Println("Error getting spotify token.", err)
+		http.Redirect(w, r, "/users/login?spotConnFailed=true", http.StatusSeeOther)
+		return
```

Comment:
With the addition of this return, we no longer need the else portion. It is automatically an "else" because the if will return. Recommend removing the else and brackets, but leave the code